helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -248,7 +248,8 @@
       "description": "Logs collection engine (deprecated, only otel is supported)",
       "type": "string",
       "enum": [
-        "otel"
+        "otel",
+        ""
```

Comment:
The only down side here is if someone uses `logsEngine: ""` in their custom, manually supplied values.yaml there won't be any deprecation message. I don't see this as much of a concern as it's now deprecated and an empty value is pretty clearly a no-op.